client/src/app/core/server/server.service.ts

```diff
@@ -58,6 +58,11 @@ export class ServerService {
       registeredExternalAuths: [],
       registeredIdAndPassAuths: []
     },
+    federation: {
+      videos: {
+        federateUnlisted: true
```

Comment:
Do you want admins to update this setting on the web interface? If no, you can remove this